Source/Scene/Primitive.js

```diff
@@ -683,6 +683,30 @@ define([
         return renamedVS + '\n' + showMain;
     };
 
+    Primitive._appendDistanceDisplayConditionToShader = function(primitive, vertexShaderSource) {
+        if (!defined(primitive._attributeLocations.distanceDisplayCondition)) {
+            return vertexShaderSource;
+        }
+
+        var renamedVS = ShaderSource.replaceMain(vertexShaderSource, 'czm_non_distanceDisplayCondition_main');
+        var distanceDisplayConditionMain =
+            'attribute vec3 boundingSphereCenter3DHigh; \n' +
+            'attribute vec3 boundingSphereCenter3DLow; \n' +
+            'attribute float boundingSphereRadius; \n' +
+            'attribute vec2 distanceDisplayCondition; \n' +
```

Comment:
I though about that, but it doesn't fit in to how per instance geometry attributes work. Each attribute must be in its own vertex buffer so no interleaving or packing. It would be a significant amount of work to change that. We really need something similar to a 3D tiles batch table like we discussed offline.